src/renderer/pages/preferences-page.js

```diff
@@ -231,6 +250,9 @@ class PreferencesPage extends React.Component {
           {this.setDefaultAppButton()}
         </PreferencesSection>
         {this.setStartupSection()}
+        <PreferencesSection title='Miscellaneous'>
```

Comment:
I think we should group the startup and sound options under a 'General' section instead.

Reply:
You're right, looks cleaner to me too 👍